### ISSUE TITLE
Batched models visualisation error

### ISSUE THREAD

**Issue by bertt** [mapbox/mapbox-gl-js#13108]
When using custom batched models (I know it's not really supported yet, but anyway...) in MapBox GL JS v3.2.0 I *sometimes* observe wrong triangle visuzualization.

![image](https://github.com/mapbox/mapbox-gl-js/assets/538812/a672bbe7-58be-4c1f-85e4-9a2d433c1f46)

Demo: https://bertt.github.io/mapbox_issues/triangles_wrong/issue/

I've got the feeling it's related to Draco compression in some geometry (couldn't isolate it more yet).

The used glTF model is visualized well in for example 3D Viewer: https://3dviewer.net/#model=https://bertt.github.io/mapbox_issues/triangles_wrong/issue/content/15-16914-10992.glb

**Comment by mourner** [comment]
Yeah, that looks bad... Might be something in the way we handle geometry, but might be good to confirm whether it's related to Draco or not by reencoding the model as non-Draco and seeing if it has the same issue.

<img width="1135" alt="image" src="https://github.com/mapbox/mapbox-gl-js/assets/25395/fd64b843-2985-4559-a5d2-813cab775e30">

**Comment by mourner** [comment]
Ah, I see you've already tried it with #13106. Will investigate what's going on there...

**Comment by bertt** [comment]
any updates on this?

**Comment by woyehenni** [comment]
any updates on this?
